README.md

```diff
@@ -19,8 +19,22 @@ This Cabal Git repository contains the following main packages:
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Ways to get the `cabal-install` binary
---------------------------------
+## Support window for releases
```

Comment:
It'd be great to add the GHC support window info somewhere here too. Because that's what I thought about first time I saw this heading.

Reply:
I recently got reminded that the GHC support window is mentioned in the contributing.md for some reason. Don't wanna litigate it further...